Cargo.toml

```diff
@@ -130,7 +130,7 @@ serde = { version = "1.0.216", features = ["derive", "rc"] }
 serde_json = { version = "1.0.133", features = ["preserve_order"] }
 similar-asserts = { version = "1.6.0" }
 smallvec = { version = "1.7.0", features = ["union"] }
-stack-graphs = { version = "0.13.0" }
+stack-graphs = { path = "submodules/stack-graphs/stack-graphs", version = "0.14.0" }
 string-interner = { version = "0.17.0", features = [
     "std",
```

Comment:
since we are forking/editing `NomicFoundation/stack-graphs`, I suggest doing a few changes there first:

- keeping the `main` branch pure for upstream changes.
- adding a `nomic` branch that contains both upstream+our changes. We can regularly merge  changes from `main` to it.
- send PR(s) to `nomic` branch with the intended changes.

This will make sure we review changes there, and that is kept up to date.
To help with this, I'm creating the `nomic` branch now, and will add CI checks/validation to it.

Reply:
Sounds good. I'll set this up.

Reply:
Here's the [PR](https://github.com/NomicFoundation/stack-graphs/pull/2). Since we're no longer using the crate as a submodule, we can also remove the linter/formatting configuration options that we had added to ignore warnings.